generators/spring-data-relational/templates/src/test/java/_package_/config/MsSqlTestContainer.java.ejs

```diff
@@ -42,6 +42,7 @@ public class MsSqlTestContainer implements SqlTestContainer {
     public void afterPropertiesSet() {
         if (null == mSSQLServerContainer) {
             mSSQLServerContainer = new MSSQLServerContainer<>("<%- dockerContainers.mssql %>")
+                .acceptLicense()
```

Comment:
```suggestion
                // You are required to accept EULA license for SQL server containers
                // Refer to https://java.testcontainers.org/modules/databases/mssqlserver/
                //.acceptLicense()
```

Reply:
Hi @mshima,

Thanks for your review! I agree with your point about not automatically accepting the license for the end user. It's indeed their responsibility to handle this. Commenting out the line seems like a reasonable approach to provide guidance while allowing users to make the decision themselves.